src/components/search/_search.scss

```diff
@@ -6,7 +6,7 @@
     flex-flow: row nowrap !important;
   }
   .ons-search__btn {
-    margin-top: -3px;
```

Comment:
was this css property not taking effect then?

Reply:
Yes - It was being overridden by the default margin styles being applied on line 12